widgets/testimonial/tpl/default.php

```diff
@@ -17,12 +17,22 @@
 				<?php endif; ?>
 
 				<div class="sow-testimonial-user">
+					<?php if( $sow_user_image = $this->testimonial_user_image($testimonial, $design) ){ ?>
```

Comment:
This would be a bit more readable if you assigned the variable before the `if` statement. E.g.
```
$sow_user_image = $this->testimonial_user_image($testimonial, $design);
if( ! empty( $sow_user_image ) )
//etc
```